pkg/agent/multicast/mcast_controller.go

```diff
@@ -298,15 +369,30 @@ func (c *Controller) syncGroup(groupKey string) error {
 		return nil
 	}
 	status := obj.(*GroupMemberStatus)
+	memberPorts := make([]uint32, 0)
```

Comment:
```suggestion
	memberPorts := make([]uint32, 0, len(status.localMembers))
```
Since it knows the maximum number already and interface not found should not be common.